Engine/Setup/BrokerageSetupHandler.cs

```diff
@@ -307,6 +307,13 @@ public bool Setup(IAlgorithm algorithm, IBrokerage brokerage, AlgorithmNodePacke
                 {
                     // populate the algorithm with the account's current holdings
                     var holdings = brokerage.GetAccountHoldings();
+
+                    // add options first to ensure raw data normalization mode is set on the equity underlyings
+                    holdings = holdings
+                        .Where(x => x.Type == SecurityType.Option)
+                        .Union(holdings.Where(x => x.Type != SecurityType.Option))
```

Comment:
maybe instead of doing multiple operations, you could take advantage of the `OrderByDescending()` targeted at the `type`? Idem for bellow

Reply:
ty, fixed